src/commands/run.ts

```diff
@@ -405,16 +405,15 @@ const fluenceRun = async (args: RunArgs) => {
   }
 
   await initFluenceClient(args, args.maybeFluenceConfig);
-  const { Fluence, callAquaFunction } = await import("@fluencelabs/js-client");
-
-  const result = await callAquaFunction({
-    args: args.runData ?? {},
-    config: {},
-    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
-    def: functionCall.funcDef,
-    peer: Fluence.getClient(),
-    script: functionCall.script,
-  });
+  const { Fluence, v5_callFunction } = await import("@fluencelabs/js-client");
+
+  const runArgs = Object.values(args.runData ?? []);
```

Comment:
These are named arguments and here you rely on order of them instead of relying on their names as it worked previously. This will break if people don't order their arguments correctly